src/gui/accountsettings.cpp

```diff
@@ -177,17 +176,22 @@ AccountSettings::AccountSettings(AccountStatePtr accountState, QWidget *parent)
     connect(FolderMan::instance(), &FolderMan::folderListChanged, _model, &FolderStatusModel::resetFolders);
     connect(this, &AccountSettings::folderChanged, _model, &FolderStatusModel::resetFolders);
 
-
-    QColor color = palette().highlight().color();
-    ui->quotaProgressBar->setStyleSheet(QString::fromLatin1(progressBarStyleC).arg(color.name()));
-
     ui->connectLabel->setText(tr("No account configured."));
 
     connect(_accountState.data(), &AccountState::stateChanged, this, &AccountSettings::slotAccountStateChanged);
     slotAccountStateChanged();
 
-    connect(&_quotaInfo, &QuotaInfo::quotaUpdated,
-        this, &AccountSettings::slotUpdateQuota);
+    if (_accountState->supportsSpaces()) {
+        ui->quotaProgressBar->setVisible(false);
```

Comment:
Maybe `hide()`? Or does it have unwanted side effects.

Reply:
I think one is just a convenience function.

Reply:
Seems so, looking at the code.